wavefront_pyformance/delta.py

```diff
@@ -22,10 +23,11 @@ def delta_counter(registry, name):
             else DeltaCounter.DELTA_PREFIX + name)
     try:
         ret_counter = DeltaCounter()
+        name = TaggedRegistry.encode_key(name, tags)
         registry.add(name, ret_counter)
         return ret_counter
     except LookupError:
-        return registry.counter(name)
+        return registry.counter(name, tags)
```

Comment:
This assumes the registry passed in is a TaggedRegistry? If this method is called with the base registry this call will raise an error even if tags is None.